lib/src/generic_utils.R

```diff
@@ -0,0 +1,11 @@
+
+#' write a unique value to a text file when this function is called.
+#' Replaces existing text in the file
+#'
+#' @param filepath a text filepath that can be written to
+#'
+#' @details this function is used to keep a dependency stale always, assuming this function is called whenever
+#' that file is referenced and there is at least a paired target use of the file.
+make_file_stale <- function(filepath){
+  cat(file = filepath, paste0(format(Sys.time(), '%m/%d/%y %H:%M:%S; random ID: '), sample(1E6, 1)))
```

Comment:
What's the purpose of the random ID (as opposed to just MM-DD-YYYY H:M:S) ?

Reply:
Thanks - I added a commit with more documentation explaining this. The random number is there because we want
to write something unique to the file, and there is a potential that this `make_file_stale()` function could be called several times in one second. Without the random ID added to it, the calls mad within the same second won't modify the file uniquely, and then it won't make the other targets stale.